bin/schema_generator.sh

```diff
@@ -133,11 +133,12 @@ function main() {
         if [[ $fname =~ metadata/sources ]] ; then
             continue
         fi
+        # This is a legacy camelCase docType and should be ignored in favor of
+        # its kebab-case replacement.
+        if [[ $fname =~ [^-_.a-z0-9] ]] ; then
```

Comment:
Is there a reason not to use `[A-Z]` here instead? This special processing is specific to camelCase and the regex encodes the snake_case special case since that doesn't require said processing.